2021q2/pmc.adoc

```diff
@@ -0,0 +1,34 @@
+== Performance Monitoring Counters
+
+Contact: Mitchell Horne <mhorne@FreeBSD.org>  
+
+This quarter, some improvements were made to the `pmc(3)` library and
+`hwpmc(4)` kernel module.
+
+Background: Traditionally, PMC event codes have been defined manually in the
+`sys/dev/hwpmc/pmc_events.h` header, and compiled statically into the kernel.
```

Comment:
IIRC, 2-item lists have no comma before "and".

Reply:
Thanks, seems to be a bad habit of mine :)